Goodlook readme (#5)
fluidos-project/quick-start-guide: README.md

Support this research by **giving it a star**. Thanks!

## Documentation

The most up-to-date documentation about the project, the overall architecture, and the current implementation of the components is available in the [Docs](https://github.com/fluidos-project/Docs) repository.

## :classical_building: The Fluidos Architecture

### Fluidos Node
<div><img src=".assets/img/fluidos-node.png" style="max-width: 50%;"/></div>

## :desktop_computer: Software components

### Currently available components

<details>
    <summary> Fluidos Node </summary>

- A component that can consist of either a single device or a set of devices, primarily serving as a representation of a Kubernetes node. It is managed by a singular Kubernetes Control Plane. For additional information, kindly consult the [FLUIDOS node repository](https://github.com/fluidos-project/node)

</details>

<details>
    <summary> Fluidos at the edge</summary>

- FLUIDOS at the edge: a minimal architecture for running the FLUIDOS components at the edge of the network on some STM boards, leveraging KubeEdge.

</details>

### Current standalone components

<details>
    <summary> Cyber Deception </summary>

> - A component to provide Cloud Native Cyber Deception as a service, thus enhancing the overall security of the FLUIDOS ecosystem. For additional information, kindly consult the [Cyber Deception repository](https://github.com/fluidos-project/cyber-deception)

Reviewer: ```suggestion
- A component to provide Cloud Native Cyber Deception as a service, thus enhancing the overall security of the FLUIDOS ecosystem. For additional information, check the [Cyber Deception repository](https://github.com/fluidos-project/cyber-deception).
```

</details>

## :computer: Click-and-play playground
This section summarizes the basic steps to reach a minimal setup of the available FLUIDOS components.

<details>

>     <summary> Fluidos Node </summary>

Reviewer: ```suggestion
    <summary> ### Fluidos Node </summary>
```

>     <summary> ### Fluidos Node </summary>

Reviewer: ```suggestion
    <summary>**Fluidos Node**</summary>
```

### Fluidos Node

> - Explore the FLUIDOS node with our step-by-step guide to effortlessly set up a FLUIDOS Node testbed using KIND (Kubernetes in Docker), which represents the simplest method to install this software on your local machine. [Begin your journey here.](https://github.com/fluidos-project/node/tree/main/testbed/kind)

Reviewer: ```suggestion
- You can set up a FLUIDOS Node testbed using KIND (Kubernetes in Docker), which represents the simplest method to install this software on your local machine. [Begin your journey here.](https://github.com/fluidos-project/node/tree/main/testbed/kind).
```

</details>

<details>
    <summary> Fluidos at the edge </summary>

### Fluidos at the edge

> - A minimal architecture for running the FLUIDOS components at the edge of the network on some STM boards, leveraging KubeEdge.

Reviewer: Missing pointer here.
